coordinator/pipeline.go

```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/hermeznetwork/tracerr"
 )
 
+// TODO: add this timeout to config file?
```

Comment:
Suggestion: remove this coment and open an issue to add this in the config later on.

Reasoning: I'm not sure if we've implemented some sort of default values vars that come from the config file. If not adding this would need to be required which would break the already deployed nodes using the config file as it is. If this is the case, and we follow `semver` we would be forced to do a major update.

If you agree on the suggestion, please in the issue mention to investigate the possibility of default values.